### ISSUE TITLE
[Bug]: Milvus standalone server triggers SIGSEGV if run within gdb

### ISSUE THREAD

**Issue by alexanderguzhva** [milvus-io/milvus#27708]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: master branch at bf46ffd6c4ec929bb8f2b1a69dd7b1d67d425eb3 from Oct, 10
- Deployment mode(standalone or cluster): standalone
- MQ type(rocksmq, pulsar or kafka): default one
- SDK version(e.g. pymilvus v2.0.0rc2): from the given branch
- OS(Ubuntu or CentOS): ubuntu 20.04
- CPU/Memory: 4 CPU, 16 GB RAM
- GPU: None
- Others:
```

### Current Behavior

I let `gdb` run `milvus` in standalone mode and I see an unexpected SIGSIGV exception inside it. Program does not crash, though  

### Expected Behavior

No exception

### Steps To Reproduce

```markdown
1. setup via `docker-compose up` in `deployments/docker/dev` directory
2. run `sudo gdb --args ./milvus run standalone`
3. run `run` in gdb, let it start
4. run `python3 -W ignore -m pytest -n 8 test_search.py` in `tests/python_client/testcases` directory
5. wait for `gdb` to get triggered
```

### Milvus Log

this is what is printed in the console
https://gist.github.com/alexanderguzhva/0986e41a4bc9bc21c7b4086d18130e92

**Comment by yanliang567** [comment]
/assign @wangting0128 
could you please help to reproduce it in house?
/unassign

**Comment by wangting0128** [comment]
I don’t have a local reproduction environment, please check.

**Comment by yanliang567** [comment]
/assign @jiaoew1991 
could you please help to take a look?
/unassign @liliu-z

**Comment by alexanderguzhva** [comment]
please feel free to ping me if you need any assistance in reproducing this problem

**Comment by jiaoew1991** [comment]
```
receiveSize := proto.Size(it.insertMsg)
rateCol.Add(internalpb.RateType_DMLInsert.String(), float64(receiveSize))
```
In the above code, insertMsg is inherited from proto.Message instead of being generated by protobuf, so it will trigger the exception logic inside protobuf. It will cause an error in debug mode.

will fixed by #27736

**Comment by xiaofan-luan** [comment]
@alexanderguzhva 
Any chance you fix the issue?

**Comment by alexanderguzhva** [comment]
@xiaofan-luan I think it got fixed in #27736

**Comment by xiaofan-luan** [comment]
close for now
